addons/api/tests/unit/serializers/storage-bucket-test.js

```diff
@@ -126,6 +159,7 @@ module('Unit | Serializer | storage bucket', function (hooks) {
           bucket_name: 'bucketname',
           bucket_prefix: 'bucketprefix',
           worker_filter: 'workerfilter',
+          plugin: { name: 'aws' },
```

Comment:
Why the need to add `plugin` in this test?

Before this PR, the serializer didn't check `pluginType` to perform a different serialization. Now we do and when `pluginType` is unknown, returns default serialization when we expect an aws `pluginType` serialization. Therefore, I add the plugin name so when checking `pluginType` it returns aws rather than unknown.